docs/using/libraries/ruby/index.rst

```diff
@@ -34,7 +34,8 @@ the ``#register_class`` method. An example of this can be found :ref:`here <appl
 
 Numbers and Booleans
 ^^^^^^^^^^^^^^^^^^^^
-Polar supports both integer and floating point numbers, as well as booleans (see :ref:`basic-types`).
+Polar supports integer and floating point real numbers, as well as booleans (see :ref:`basic-types`).
+These map to the Ruby ``Integer``, ``Float``, ``TrueClass``, and ``FalseClass`` types.
```

Comment:
nit: I think this mirrors the "integer and floating point real numbers, as well as booleans" line above

```suggestion
These map to the Ruby ``Integer``, ``Float``, and ``TrueClass``/``FalseClass`` types.
```

Reply:
> nit: I think this mirrors the "integer and floating point real numbers, as well as booleans" line above

Sorry, what line where?

Reply:
Line 37: integer and floating point real numbers, as well as booleans
Line 38: These map to the Ruby `Integer`, `Float`, and `TrueClass`/`FalseClass` types.